test/integration/acceptance/get_roles_test.cpp

```diff
@@ -4,53 +4,46 @@
  */
 
 #include <gtest/gtest.h>
-#include "integration/acceptance/acceptance_fixture.hpp"
-#include "framework/integration_framework/integration_test_framework.hpp"
 #include "backend/protobuf/transaction.hpp"
+#include "framework/integration_framework/integration_test_framework.hpp"
 #include "framework/specified_visitor.hpp"
+#include "integration/acceptance/acceptance_fixture.hpp"
 #include "interfaces/permissions.hpp"
 
-
 using namespace integration_framework;
 using namespace shared_model;
 
-class GetRoles : public AcceptanceFixture {
-public:
-//    auto makeUserWithPerms(const interface::RolePermissionSet &perms = {
-//            interface::permissions::Role::kGetRoles}) {
-//        return AcceptanceFixture::makeUserWithPerms(perms);
-//    }
-
-};
+class GetRoles : public AcceptanceFixture {};
 
 /**
  * @given a user with CanGetRoles permission
  * @when execute query with getRoles command
  * @then there is should be no exception
  */
 TEST_F(GetRoles, CanGetRoles) {
-    auto checkQuery = [](auto &queryResponse) {
-        ASSERT_NO_THROW(boost::apply_visitor(
-                framework::SpecifiedVisitor<shared_model::interface::RolesResponse>(), queryResponse.get()));
-    };
+  auto checkQuery = [](auto &queryResponse) {
```

Comment:
Rewrite "then" section of the docstring. It should indicate that the list of roles can be received.
Presence or missing of an exception is an implementation detail of the test and should not be mentioned in the docstring.